modules/Users/Users.php

```diff
@@ -527,11 +527,36 @@ public function change_password($user_password, $new_password, $dieOnError = tru
 			$this->error_string = $mod_strings['ERR_PASSWORD_INCORRECT_OLD'];
 			return false;
 		}
-
 		//set new password
 		$crypt_type = $this->DEFAULT_PASSWORD_CRYPT_TYPE;
 		$encrypted_new_password = $this->encrypt_password($new_password, $crypt_type);
 
+		global $adb;
+		require_once 'modules/Emails/Emails.php';
+
+		$emailObj = new Emails;
+
+		$par_id = $this->id;
+
+		$to_email = $this->column_fields['email1'];
+
+		$templateName = 'Password Change Template';
+
+		$subject = getTranslatedString('MSG_SUB');
+
+		$context = array(
+			'$subject$' => $subject,
+			'$user_name$'=> $usr_name,
+			'$user_password$'=> $new_password
+		);
+
+		$module = 'Users';
+
+		$emailObj->sendEmailTemplate($templateName, $context, $module, $to_email, $par_id);
```

Comment:
change to:  Emails::sendEmailTemplate(..
also if you can eliminate some variables and use direct values in the call better, for example, there is no need for $par_id nor $module